### PR TITLE
[zfs] Add plugin for Ubuntu/Debian

### PR DESCRIPTION
**Pull request by slashdd**
Add ZFS plugin for Ubuntu to show pool information if zfsutils-linux package is present.

### CODE REVIEW

**Comment by slashdd** [comment]
Output example  
--
$ cat <path_to_sosreport>/sos_commands/zfs/zpool_list
NAME        SIZE  ALLOC   FREE  EXPANDSZ   FRAG    CAP  DEDUP  HEALTH  ALTROOT
pool-test  9.94G  62.5K  9.94G         -     0%     0%  1.00x  ONLINE  -

$ cat <path_to_sosreport>/sos_commands/zfs/zpool_status_-x
all pools are healthy
--

**Comment by bmr-cymru** [comment]
Looks good from a quick scan.

I think both Debian and SuSE are also shipping or enabling ZFS (at least for OpenSuSE), so it would make sense to also enable this plugin for those distros.

**Comment by slashdd** [comment]
@bmr-cymru sure I can enabled Debian which will be easy since it uses the same binary package name, and I can also create another class for SusePlugin.

Would you like me to append this existing PR to include Debian/Suse ? or do another one ?

**Comment by slashdd** [comment]
@bmr-cymru, I have included DebianPlugin, so the current PR supports Ubuntu/Debian. I'm not familiar SuSE ZFS as we speak, I will need some time to test. May I suggest to include the SuSE plugin in a near future PR ?

**Comment by bmr-cymru** [comment]
Sure, no problem - thanks for updating this PR. I'll try to get it included in the next batch of merges.

File: sos/plugins/zfs.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+
+from sos.plugins import Plugin, UbuntuPlugin, DebianPlugin
+
+
+class DebianZfs(Plugin, UbuntuPlugin, DebianPlugin):
```
Comment (bmr-cymru) [inline sos/plugins/zfs.py]: The class name should just be `Zfs`, but I'll fix that up when I merge.

**Comment by bmr-cymru** [comment]
Missed this earlier - your commit is missing a s"Signed-off-by", required by the [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines). You can update the commit with `git commit --amend` and re-push it with `git push --force`, or if you give your permission I can add it when I merge the patch.

**Comment by slashdd** [comment]
Done - "Signed-off-by" amended and pushed. 

Thanks for reviewing it.

**Comment by bmr-cymru** [comment]
Thanks for the quick turn-around! Should make 3.4.

**Pull request closed without merging**